src/server/game/Spells/Spell.cpp

```diff
@@ -4739,6 +4739,8 @@ SpellCastResult Spell::CheckCast(bool strict)
         {
             if (m_triggeredByAuraSpell)
                 return SPELL_FAILED_DONT_REPORT;
+            else if (m_spellInfo->HasAttribute(SPELL_ATTR0_DISABLED_WHILE_ACTIVE))  // Shadowform, Tricks of the Trade, Misdirection
```

Comment:
tested it now - it will anyway work fine without this part 
the only part I m missing now is why do tricks of the trade and misdirection not work with the same piece of code - I figure I might have dome some other mods unrelated to this in the past, I'm figuring it out now